texastribune/styleguides
# Add the TTF14 style guide

#### What's the plan?

This PR adds in a style guide for the TTF14 site.
#### What's this PR do?

Adds a TTF14 style guide HTML page and styles, particularly TTF14festivalstyles.html.

Note: When this PR and the other that switches all styles to Sass are both merged in, then festival.css won't need to be checked in. It's in here for testing purposes to ensure everything looks good before they're merged together.
#### How should this be manually tested?

Run locally with:
- `grunt` to build css
- `make serve`

Navigate to the TTF14 styleguide and click around. Read through for completeness of content and the best wording of copy. Check that everything looks good. Try out the Example Code by clicking to toggle it to display and not display.
#### TODO
- [x] Check in TTF14 css
#### Smells?

Share any you see
